src/utils/scoreCalculator.ts

```diff
@@ -0,0 +1,20 @@
+import { AsyncAPIDocumentInterface } from '@asyncapi/parser/cjs/models';
+
+export async function calculateScore(document: AsyncAPIDocumentInterface | undefined) {
+  let scoreEvaluate = 0;
+  if (document?.info().hasDescription()) {
+    scoreEvaluate += 0.15;
+  }
+  if (document?.info().hasLicense()) {
+    scoreEvaluate += 0.25;
+  }
+  if (!document?.servers().isEmpty()) {
+    scoreEvaluate += 0.25;
+  }
+  if (!document?.channels().isEmpty()) {
+    scoreEvaluate += 0.35;
+  }
+  const finalScore = (scoreEvaluate / 1) * 100;
+  
+  return `The score of the asyncapi document is ${finalScore}`;
```

Comment:
How about returning the score instead of a log string. As this makes it easier to use later in another scenarios if we want.

Reply:
@Shurtu-gal In my opinion, proper logging would be better that enhanve the developer experience as well. @Amzani What do you think on this ?

Reply:
Yeah, I am not suggesting to take out the log message, just return the score from this function. And put logging where this function is consumed.

This way this function can be reused if needed.